### PR TITLE
DOCSUP-7363: Documentation of the 'file()' function

### PR DESCRIPTION
**Pull request by kirillikoff**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category:

- Documentation for #19204

### CODE REVIEW

File: docs/en/sql-reference/functions/files.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+**Syntax**
+
+``` sql
+file(path)
+```
+
+**Arguments**
+
+-   `path` — The relative path to the file from [user_files_path](../../operations/server-configuration-parameters/settings.md#server_configuration_parameters-user_files_path). Path to file support following wildcards in read-only mode: `*`, `?`, `{abc,def}` and `{N..M}` where `N`, `M` — numbers, `'abc', 'def'` — strings.
```
Comment (olgarev) [inline docs/en/sql-reference/functions/files.md:20]:
```suggestion
-   `path` — The relative path to the file from [user_files_path](../../operations/server-configuration-parameters/settings.md#server_configuration_parameters-user_files_path). Path to file support following wildcards: `*`, `?`, `{abc,def}` and `{N..M}` where `N`, `M` — numbers, `'abc', 'def'` — strings.
```

File: docs/ru/sql-reference/functions/files.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+**Синтаксис**
+
+``` sql
+file(path)
+```
+
+**Аргументы**
+
+-   `path` — относительный путь до файла от [user_files_path](../../sql-reference/table-functions/file.md#server_configuration_parameters-user_files_path). Путь к файлу поддерживает следующие шаблоны в режиме доступа только для чтения `*`, `?`, `{abc,def}` и `{N..M}`, где `N`, `M` — числа, `'abc', 'def'` — строки.
```
Comment (olgarev) [inline docs/ru/sql-reference/functions/files.md:20]:
```suggestion
-   `path` — относительный путь до файла от [user_files_path](../../sql-reference/table-functions/file.md#server_configuration_parameters-user_files_path). Путь к файлу поддерживает следующие символы подстановки и шаблоны: `*`, `?`, `{abc,def}` и `{N..M}`, где `N`, `M` — числа, `'abc', 'def'` — строки.
```

File: docs/en/sql-reference/functions/files.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+**Example**
+
+Inserting data from files a.txt and b.txt into a table as different rows.
```
Comment (olgarev) [inline docs/en/sql-reference/functions/files.md:24]:
```suggestion
Inserting data from files a.txt and b.txt into a table:
```

File: docs/en/sql-reference/functions/files.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
+---
+toc_priority: 43
+toc_title: Files
+---
+
+# Functions for Working with Files {#functions-for-working-with-files}
+
+## file {#file}
+
+Reads file as a String. The file can contain subqueries, conditions, names of columns and any other information that will be read as one line.
```
Comment (olgarev) [inline docs/en/sql-reference/functions/files.md:10]:
```suggestion
Reads file as a String. The file content is not parsed, so any information will be read as one line.
```

File: docs/ru/sql-reference/functions/files.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
+---
+toc_priority: 43
+toc_title: "Функции для работы с файлами"
+---
+
+# Функции для работы с файлами {#funktsii-dlia-raboty-s-failami}
+
+## file {#file}
+
+Читает файл как строку. Файл может содержать подзапросы, условия, названия столбцов и любую другую информацию, которая будет прочитана как одна строка.
```
Comment (olgarev) [inline docs/ru/sql-reference/functions/files.md:10]:
```suggestion
Читает файл как строку. Содержимое файла не разбирается (не парсится) и записывается в указанную колонку в виде единой строки.
```

**Review by nikitamikhaylov — APPROVED** [review]

**Comment by olgarev** [comment]
@nikitamikhaylov  Hi, will you merge this PR, pls? So we can close the ticket.

**Pull request merged**